starter-code/index.js

```diff
@@ -1,10 +1,57 @@
-var SortedList = function() {};
+var SortedList = function() {
+  this.items = [];
+  this.length = 0;
+};
+
+
+
+
+
+SortedList.prototype.add     = function(item) {
+  this.items.push(item);
+  this.items.sort(function(a,b){return a-b});
+  this.length = this.items.length;
+
+
+
+}
+SortedList.prototype.get = function(pos) {
+return this.items[pos - 1];
+}
+
+SortedList.prototype.max = function() {
+  return Math.max.apply(null, this.items);
+
+}
+
+SortedList.prototype.min = function() {
+  return Math.min.apply(null, this.items);
```

Comment:
Very clever using the `.apply` method here. Just so you can compare, here was another way to write this method:
```js
  var minNumber =
    this.list.reduce((minSoFar, number) => {
      return Math.min(minSoFar, number);
    });
  return minNumber;
};
```